Mathlib/Topology/Compactification/OnePointEquiv.lean

```diff
@@ -0,0 +1,166 @@
+/-
+Copyright (c) 2024 Bjørn Kjos-Hanssen. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Bjørn Kjos-Hanssen
+-/
+import Mathlib.Data.Real.Basic
+import Mathlib.LinearAlgebra.Projectivization.Basic
+import Mathlib.Topology.Compactification.OnePoint
+
+/-!
+# OnePoint Compactification and Projectivization
+
+We construct a set-theoretic equivalence between
+`OnePoint K` and the projectivization `ℙ K (Fin 2 → K)`
+for an arbitrary field `K`.
+
+(This equivalence can be extended to a homeomorphism
+in the case `K = ℝ`, where `OnePoint ℝ` gets the topology of one-point compactification.
+This result is to be added in a different file.)
+
+## Main definitions
+
+* `divOnePoint`: Division on `K` taking values in `OnePoint K`.
+* `divSlope`: Lifting `divOnePoint` to a map `ℙ K (Fin 2 → K) → OnePoint K`.
+
+## Main results
+
+* `ℙ K (Fin 2 → K) ≃ OnePoint K`
+
+## Tags
+
+one-point extension, projectivization
+-/
+
+
+open scoped LinearAlgebra.Projectivization OnePoint
+open Projectivization Classical
+
+/-!
+### Definition and basic properties
+
+In this section we define `divOnePoint` and prove that it lifts to `divSlope`.
```

Comment:
At this point, the reader has not seen `divOnePoint` and `divSlope` - can you tweak this explanation, please?